### ISSUE TITLE
Partial traversal using A*

### ISSUE THREAD

**Issue by froozen** [Jugendhackt/paketmagie#21]
Instead of having the algorithm traverse the whole tree, we'll use A\* to pick the most promising states and continue from those until we hit the destination node.

**Comment by sternenseemann** [comment]
There are still some issues to be resolved by @froozen. Waiting.

**Comment by fkarg** [comment]
is the A*-branch expected to get merged soon?

**Comment by fkarg** [comment]
or rather, our improvements even without the Dijkstra to begin with

**Comment by sternenseemann** [comment]
No, @froozen has to resolve a few issues with the implementation, I believe.
